etc/notes/CHANGES_7.0.0.md

```diff
@@ -0,0 +1,248 @@
+
+### `bson` and `mongodb-connection-string-url` versions 7.0.0
+
+This driver version has been updated to use `bson@7.0.0` and `mongodb-connection-string-url@7.0.0`, which match the driver's Node.js runtime version support. BSON functionality re-exported from the driver is furthermore subject to the changes outlined in the [BSON V7 release notes](https://github.com/mongodb/js-bson/releases/tag/v7.0.0).
+
+### Optional peer dependency releases and version bumps
+
+- `@mongodb-js/zstd` optional peer dependency minimum version raised to `7.0.0`, dropped support for `1.x` and `2.x` (note that `@mongodb-js/zstd` does not have `3.x-6.x` version releases)
+- `kerberos` optional peer dependency minimum version raised to `7.0.0`, dropped support for `2.x` (note that `kerberos` does not have `3.x-6.x` version releases)
+- `mongodb-client-encryption` optional peer dependency minimum version raised to `7.0.0`, dropped support for `6.x`
+
+Additionally, the driver is now compatible with the following packages:
+
+| Dependency                    | Allowed Range | Previous Range |
+| ----------------------------- | ------------- | -------------- |
+| @aws-sdk/credential-providers | ^3.806.0      | ^3.188.0       |
+| gcp-metadata                  | ^7.0.1        | ^5.2.0         |
+| socks                         | ^2.8.6        | ^2.7.1         |
+
+## 🔐 AWS authentication
+
+To improve long-term maintainability and ensure compatibility with AWS updates, we’ve standardized AWS auth to use the official SDK in all cases and made a number of supporting changes outlined below.
+
+### `@aws-sdk/credential-providers` is now required for MONGODB-AWS authentication
+
+Previous versions of the driver contained two implementations for AWS authentication and could run the risk of the custom driver implementation not supporting all AWS authentication features as well as not being correct when AWS makes changes. Using the official AWS SDK in all cases alleviates these issues.
+
+```bash
+npm install @aws-sdk/credential-providers
+```
+
+### Custom AWS credential provider takes highest precedence
+
+When providing a custom AWS credential provider via the auth mechanism property `AWS_CREDENTIAL_PROVIDER`, it will now take the highest precedence over any other AWS auth method.
+
+### Explicitly provided credentials no longer accepted with MONGODB-AWS authentication
+
+AWS environments (such as AWS Lambda) do not have credentials that are permanent and expire within a set amount of time. Providing credentials in the URI or options would mandate that those credentials would be valid for the life of the `MongoClient`, which is problematic. With this change, the installed required AWS SDK will now fetch credentials using the environment, endpoints, or a custom credential provider.
```

Comment:
```suggestion
AWS environments (such as AWS Lambda) do not have credentials that are permanent and expire within a set amount of time. Providing credentials in the URI or options would mandate that those credentials would be valid for the life of the `MongoClient`, which is problematic. With this change, the installed required AWS SDK will now fetch credentials using a custom credential provider or the AWS SDK.
```

Suggest removing all mention of what the SDK supports.

Reply:
rephrased, let me know what you think